packages/extension/src/companion/CompanionPopupButton.tsx

```diff
@@ -61,12 +64,13 @@ export const CompanionPopupButton = (): ReactElement => {
     >
       <Button
         onClick={onButtonClick}
-        className={classNames(
-          'mr-4 border-theme-status-cabbage hidden laptop:flex',
+        variant={
           showCompanionPermission
-            ? 'btn-primary-cabbage'
-            : 'btn-secondary-cabbage',
-        )}
+            ? ButtonVariant.Primary
+            : ButtonVariant.Secondary
+        }
+        color={ButtonColor.Cabbage}
+        className="hidden laptop:flex mr-4 border-theme-status-cabbage"
```

Comment:
Was this something custom, `border-theme-status-cabbage` that the button didn't have by default or props?

Reply:
For secondary button type, by default we show a white border on dark theme. This changes the border to the cabbage color. So it is custom only for this button.